samcli/lib/utils/temp_file_utils.py

```diff
@@ -2,25 +2,4 @@
 Helper functions for temporary files
 """
 import os
```

Comment:
I moved these to `osutils.py` to consolidate all OS related helpers. This is not required for this PR, but I thought it might be a good cleanup to do while I am here